boards/rt-thread/art_pi2/board.yml

```
@@ -0,0 +1,6 @@
+board:
+  name: art_pi2
+  full_name: RT-Thread ART-Pi2
```

```suggestion
  full_name: ART-Pi2
```

What's more, I cannot find any reference to this board online -- is this really how the board is named or is this a "rev. 2" of the ART-Pi?

---

I prefer taking this as separate board named "ART-Pi2", distinct from ART-Pi with different mcu and other features. 

Documentation can be found here: [sdk-bsp-stm32h7r-realthread-artpi2](https://github.com/RT-Thread-Studio/sdk-bsp-stm32h7r-realthread-artpi2)
Schematics are available here: [schematics](https://github.com/RT-Thread-Studio/sdk-bsp-stm32h7r-realthread-artpi2/tree/master/documents/board/STM32H7)

For clarity:
- ART-Pi2:
  - New board featuring STM32H7R7 (this PR)
  - Just released in February, 2025, so there is not much reference online
  - Will be available on AliExpress and Mouser soon
- ART-Pi:
  - Different board featuring STM32H750, released in 2020
  - Available on [AliExpress](https://www.aliexpress.com/i/1005006020273744.html) and [Mouser](https://www.mouser.com/ProductDetail/RT-Thread/ART-PI-STM32H7?qs=sGAEpiMZZMurtJ7VwBTl0R0dDrgG59QLfiEUCT%252BwE2HEwjeA%252BAxHlg%3D%3D)

---

Great, thanks for clarifying!